spec/features/restaurants/visitor_tries_update_restaurant_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+feature 'prevent unuathorized updates to restaurant' do
+  context 'as a visitor' do
+    scenario 'I cannot save a restaurant' do
+      visit new_restaurant_path
+      expect(page).to have_content("Log in")
```

Comment:
This is to test that the model forces authentication if you try to navigate the browser to specific paths (Spencer suggested this test in a live code for rescue mission).